addons/armaos/functions/fnc_terminal_updateBatteryStatus.sqf

```diff
@@ -48,10 +48,23 @@ _handle =
                 };
 
                 _batteryCtrl ctrlSetText format ["\z\ae3\addons\armaos\images\AE3_battery_%1_percent.paa", _value];
+
+                /* ------------- UI on Texture ------------ */
+
+                if (AE3_UiOnTexture) then
```

Comment:
Maybe only push an update, when the symbol changed

Reply:
Yes, you are right. Will try to implement that.